### ISSUE TITLE
[Bug]: play() request error

### ISSUE THREAD

**Issue by shadycloud** [MarmadileManteater/FreeTubeCordova#231]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. Open freetube version 0.19.1 nightly 380 (or 371) beta using samsung galaxy A13 running android 13.
2. Trying to play 3 different videos. 
3. See error ```play() request error``` in attached screen shot.

### Expected Behavior

Video playback with audio. Feature works as it should with my environment using v0.18.0.107 Beta.
![Screenshot_20231012_220508_FreeTube Nightly](https://github.com/MarmadileManteater/FreeTubeCordova/assets/13189052/931de4e7-b699-458b-808f-d415403c2fdc)

### Issue Labels

content not loading, feature stopped working

### FreeTube Version

0.19.1 nightly 380

### Operating System Version

Android 13 

### Installation Method

.apk

### Primary API used

Local API

### Last Known Working FreeTube Version (If Any)

v0.18.0.107 Beta

### Additional Information

Click to copy also doesn't work with my setup, but I'll throw another issue up for it. 
![Screenshot_20231012_220853_Settings](https://github.com/MarmadileManteater/FreeTubeCordova/assets/13189052/ab335e42-8ced-4674-81d3-dfb7b94335a1)

### Nightly Build

- [X] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by MarmadileManteater** [comment]
What invidious instance is selected in your settings?

Currently, there is a known issue with invidious instances getting IP blocked by YouTube (see FreeTubeApp#3892), and this results in videos being unable to load while `proxyVideos` is enabled, and in FreeTubeCordova, an issue with CORS forces the `proxyVideos` setting on (see #5).

**Comment by shadycloud** [comment]
![Screenshot_20231013_183433_FreeTube Nightly](https://github.com/MarmadileManteater/FreeTubeCordova/assets/13189052/83f6bc30-782d-4cd9-80cd-678ade9aee87)

**Comment by shadycloud** [comment]
Invidious instance: https://iv.nboeck.de

Changing the setting allowed video to play. The function that should allow 'click to copy' doesn't seem to work. I've searched upstream... I don't know if that's even worth adding to the pipeline of issues though. 

![Screenshot_20231013_183433_FreeTube Nightly](https://github.com/MarmadileManteater/FreeTubeCordova/assets/13189052/3f76fe13-647a-4eaf-93a4-4348ca11c167)

**Comment by shadycloud** [comment]
Updated my instance with https://iv.ggtyler.dev from my previous 0.18 install and no more issues. I suppose I need to figure out how to generate a new address if it happens again.
